src/main/webapp/i18n/de/global.json

```
@@ -174,7 +174,12 @@
             "dev": "Development",
             "test": "Testserver"
         },
-        "item-count": "Ergebnis {{first}} - {{second}} von {{total}} Elemente."
+        "item-count": "Ergebnis {{first}} - {{second}} von {{total}} Elemente.",
```

Should "showing" be translated to something like "Zeigt"? 🤔 Not sure what would sound best ...

```suggestion
        "item-count": "Zeigt {{first}} - {{second}} von {{total}} Elementen.",
```
or maybe

```suggestion
        "item-count": "Es werden {{first}} - {{second}} von {{total}} Elementen angezeigt.",
```

---

I didn't actually change this, but yeah that translation is terrible, I went with `Elemente {{first}} - {{second}} von {{total}} werden angezeigt.` as a better translation.